src/svg/rvt-icon-eye.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+  <title>Eye Icon</title>
+  <g>
```

Comment:
You'll need to move the `fill="currentColor"` attribute up to this `<g>` tag so that every element inside will inherit the current CSS `color` property.

Currently, the dot in the middle of the eye (`<circle>` tag) will not changes color if a different CSS color value is applied.